docs/src/tutorials/pino_ode.md

```diff
@@ -0,0 +1,66 @@
+# Physics Informed Neural Operator for ODEs Solvers
+
+This tutorial provides an example of how to use the Physics Informed Neural Operator (PINO) for solving a family of parametric ordinary differential equations (ODEs).
+
+## Operator Learning  for a family of parametric ODE.
+
+In this section, we will define a parametric ODE and solve it using a PINO. The PINO will be trained to learn the mapping from the parameters of the ODE to its solution.
+
+```@example pino
+using Test
+using OrdinaryDiffEq, OptimizationOptimisers
+using Lux
+using Statistics, Random
+using NeuralPDE
+
+equation = (u, p, t) -> cos(p * t)
+tspan = (0.0f0, 1.0f0)
+u0 = 1.0f0
+prob = ODEProblem(equation, u0, tspan)
+
+# Define the architecture of the neural network that will be used as the PINO.
+branch = Lux.Chain(
+    Lux.Dense(1, 10, Lux.tanh_fast),
+    Lux.Dense(10, 10, Lux.tanh_fast),
+    Lux.Dense(10, 10))
+trunk = Lux.Chain(
+    Lux.Dense(1, 10, Lux.tanh_fast),
+    Lux.Dense(10, 10, Lux.tanh_fast),
+    Lux.Dense(10, 10, Lux.tanh_fast))
+deeponet = NeuralPDE.DeepONet(branch, trunk; linear = nothing)
+
+bounds = (p = [0.1f0, pi],)
+db = (bounds.p[2] - bounds.p[1]) / 50
+dt = (tspan[2] - tspan[1]) / 40
+strategy = NeuralPDE.GridTraining([db, dt])
+opt = OptimizationOptimisers.Adam(0.03)
+alg = NeuralPDE.PINOODE(deeponet, opt, bounds; strategy = strategy)
+sol = solve(prob, alg, verbose = false, maxiters = 2000)
+predict = sol.u
+```
+
+Now let's compare the prediction from the learned operator with the ground truth solution which is obtained by analytic solution the parametric ODE. Where 
+Compare prediction with ground truth.
+
+```@example pino
+using Plots
+# Compute the ground truth solution for each parameter
+ground_analytic = (u0, p, t) -> u0 + sin(p * t) / (p)
+p_ = bounds.p[1]:strategy.dx[1]:bounds.p[2]
```

Comment:
Not using grid training will make this more compelling since it should predict at new parameters, not ones trained on

Reply:
yes,I agree